dali/operators/image/color/color_twist.cc

```diff
@@ -143,15 +152,23 @@ bool ColorTwistCpu::SetupImpl(std::vector<OutputDesc> &output_desc, const HostWo
   const auto &input = ws.template Input<CPUBackend>(0);
   output_desc.resize(1);
   DetermineTransformation(ws);
+  auto sh = input.shape();
+  auto num_dims = sh.sample_dim();
+  auto layout = input.GetLayout();
+  int c_dim = layout.find('C');
+  DALI_ENFORCE(c_dim == num_dims - 1 || layout.empty(), make_string("Only channel last or empty "
+              "layouts are supported, received ", layout, " instead"));
+  DALI_ENFORCE(num_dims >= 3 && num_dims <= 4, make_string("Only 3 and 4 dimensions are "
+              "supported received ", num_dims));
```

Comment:
This check should be covered by the input layout defined in the schema. Here we can do an assert maybe.

Reply:
Done